content/en/docs/tasks/security/authorization/authz-custom/index.md

```diff
@@ -7,11 +7,11 @@ owner: istio/wg-security-maintainers
 test: yes
 ---
 
-This task shows you how to set up an Istio authorization policy using a new experimental value for the [action field](/docs/reference/config/security/authorization-policy/#AuthorizationPolicy-Action), `CUSTOM`,
+This task shows you how to set up an Istio authorization policy using a new value for the [action field](/docs/reference/config/security/authorization-policy/#AuthorizationPolicy-Action), `CUSTOM`,
 to delegate the access control to an external authorization system. This can be used to integrate with [OPA authorization](https://www.openpolicyagent.org/docs/latest/envoy-introduction/),
 [`oauth2-proxy`](https://github.com/oauth2-proxy/oauth2-proxy), your own custom external authorization server and more.
 
-{{< boilerplate experimental-feature-warning >}}
+This feature is currently in alpha stage.
```

Comment:
I only found two other places (maybe there are more) where we mention a feature as Alpha, and we have that in a warning. I'm not sure if we just don't label all Alpha features or if only a few are documented and those have warnings.

Reply:
removed the sentence, I guess we don't need to call out the alpha stage unless it's really needed (I'm fine either way but I don't feel there is much warning needed for this feature).